<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintImageDialog</class>
 <widget class="QDialog" name="PrintImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Image</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout" stretch="0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="XmlUi::GradientWidget" name="promptWidget" native="true">
     <layout class="QHBoxLayout" name="promptLayout" stretch="0,1">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="promptPixmap"/>
      </item>
      <item>
       <widget class="QLabel" name="promptLabel">
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="a0_radioButton">
        <property name="text">
         <string>A0 poster (14032 x 9920)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a1_radioButton">
        <property name="text">
         <string>A1 poster (9920 x7016)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a2_radioButton">
        <property name="text">
         <string>A2 poster (7016 x 4960)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a3_radioButton">
        <property name="text">
         <string>A3 paper (4960 x 3508)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a4_radioButton">
        <property name="text">
         <string>A4 paper (3508 x 2480)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="own_radioButton">
        <property name="text">
         <string>Own resolution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Image Resolution</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="spacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Width:</string>
           </property>
           <property name="buddy">
            <cstring>spinWidth</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinWidth">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>8000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Height:</string>
           </property>
           <property name="buddy">
            <cstring>spinHeight</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinHeight">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>8000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="spacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XmlUi::GradientWidget</class>
   <extends>QWidget</extends>
   <header>xmlui/gradientwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinWidth</tabstop>
  <tabstop>spinHeight</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrintImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrintImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
